recipes/libxml2/all/conanfile.py

```diff
@@ -89,18 +88,18 @@ def requirements(self):
         if self.options.iconv:
             self.requires("libiconv/1.16")
         if self.options.icu:
-            self.requires("icu/68.2")
+            self.requires("icu/69.1")
 
     def build_requirements(self):
         if not self._is_msvc:
             if self.options.zlib or self.options.lzma or self.options.icu:
-                self.build_requires("pkgconf/1.7.3")
+                self.build_requires("pkgconf/1.7.4")
             if tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH"):
-                self.build_requires("msys2/20200517")
+                self.build_requires("msys2/cci.latest")
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
-        os.rename("libxml2-{0}".format(self.version), self._source_subfolder)
+        tools.rename("libxml2-{}".format(self.version), self._source_subfolder)
```

Comment:
is possible to add a comment explaining why we cannot use the usual strip_root+destination pattern ?

Reply:
sure, I'll do